### PR TITLE
Update standings.component.html

### PR DESCRIPTION
**Pull request by Synenkyi**
### Summary

_Describe main changes going into those pull requests._

_Use "closes #(issue number)" to enable GitHub to close issue when PR is merged_

### Areas Of Interest

_Any specific places you want to bring attention of the reviewers? Some places you are uncertain in/some design decisions. (If none remove this section)_

#### Design

- [ ] Web API layer contains as little logic as possible
- [ ] Domain objects use semantic names

#### Perfromance

- [ ] In case of any DB query was changed: attach generated SQL for review
- [ ] Number of DB roundtrips should be as low as possible

#### Unit Tests

- [ ] Naming: Initial state and expected result are properly stated
- [ ] Test follows [AAA](https://medium.com/@pjbgf/title-testing-code-ocd-and-the-aaa-pattern-df453975ab80) pattern.
- [ ] Test is Isolated
- [ ] Mock instances and expected instances do not share references
- [ ] When new property is added all related comparers are updated

### CODE REVIEW

File: src/VolleyManagement.WebClient/spa/app/Components/Standings/standings.component.html [hunk 1]
```diff
@@ -63,4 +63,4 @@
             </tbody>
         </table>
     </div>
-</div>
```
Comment (YuriiChornyi) [inline src/VolleyManagement.WebClient/spa/app/Components/Standings/standings.component.html]: красавчик

Comment (sdiachen) [reply src/VolleyManagement.WebClient/spa/app/Components/Standings/standings.component.html]: @YuriiChornyi please be warned that we have [Code of Conduct](https://github.com/VolleyManagement/volley-management/blob/master/CODE_OF_CONDUCT.md)

Comment (YuriiChornyi) [reply src/VolleyManagement.WebClient/spa/app/Components/Standings/standings.component.html]: @Synenkyi,
@sdiachen Sorry 💯

**Comment by sdiachen** [comment]
@Synenkyi I think you are right in your arguments. P.S. changes will appear on production a little bit later

**Pull request merged**